sysvar/src/lib.rs

```diff
@@ -188,16 +200,9 @@ pub trait Sysvar:
         bincode::serialize_into(&mut account_info.data.borrow_mut()[..], self).ok()
     }
 
-    /// Load the sysvar directly from the runtime.
-    ///
-    /// This is the preferred way to load a sysvar. Calling this method does not
-    /// incur any deserialization overhead, and does not require the sysvar
-    /// account to be passed to the program.
-    ///
-    /// Not all sysvars support this method. If not, it returns
-    /// [`ProgramError::UnsupportedSysvar`].
+    /// Calls [`Sysvar::get`].
```

Comment:
More out of curiosity, is this `get` still needed on the `SysvarSerialize` trait? Since the implementation will have to provide the `get` due to the `Sysvar` bound, seems like it is redundant and potentially error prone (?) – e.g., technically it can be overridden and implemented differently than `Sysvar::get`.

Reply:
It doesn't seem needed, but I assumed it was included to make upgrading to v3 easier. Users can change their import from `Sysvar` to `SysvarSerialize` and recover exactly what they had before.

On the other hand, it isn't too much worse to tell people "use `SysvarSerialize` with `Sysvar`" instead of "use `SysvarSerialize` instead of `Sysvar`", so I'm good with removing, unless there's something I'm missing.

Reply:
Yeah, I can't see the need for it. I would expect that in most cases it will still be "use `SysvarSerialize` instead of `Sysvar`" to load a sysvar from an account info as it might be unlikely to be using both at the same time (`from_account_info` and `get`). For users using only `get`, there is no change.